lutris/game_actions.py

```diff
@@ -68,154 +72,179 @@ def on_game_stop(self, *_args):
+                    "install_dlcs": game.is_updatable,
+                    "stop": self.is_game_running,
+                    "configure": bool(game.is_installed),
+                    "browse": game.is_installed and game.runner_name != "browser",
+                    "show_logs": game.is_installed,
+                    "category": True,
+                    "favorite": not game.is_favorite,
+                    "deletefavorite": game.is_favorite,
+                    "install_more": not game.service and game.is_installed,
+                    "execute-script": bool(
+                        game.is_installed and game.runner
+                        and game.runner.system_config.get("manual_command")
+                    ),
+                    "desktop-shortcut": (
+                        game.is_installed
+                        and not xdgshortcuts.desktop_launcher_exists(game.slug, game.id)
+                    ),
+                    "menu-shortcut": (
+                        game.is_installed
+                        and not xdgshortcuts.menu_launcher_exists(game.slug, game.id)
+                    ),
+                    "steam-shortcut": (
+                        game.is_installed
+                        and not has_steam_shortcut
+                        and not is_steam_game
+                    ),
+                    "rm-desktop-shortcut": bool(
+                        game.is_installed
+                        and xdgshortcuts.desktop_launcher_exists(game.slug, game.id)
+                    ),
+                    "rm-menu-shortcut": bool(
+                        game.is_installed
+                        and xdgshortcuts.menu_launcher_exists(game.slug, game.id)
+                    ),
+                    "rm-steam-shortcut": bool(
+                        game.is_installed
+                        and has_steam_shortcut
+                        and not is_steam_game
+                    ),
+                    "remove": self.is_game_removable,
+                    "view": True,
+                    "hide": game.is_installed and not game.is_hidden,
+                    "unhide": game.is_hidden,
+                }
 
     def on_game_launch(self, *_args):
         """Launch a game"""
-        self.game.launch(self.window)
+        for game in self.games:
+            game.launch(self.window)
 
     def get_running_game(self):
-        if self.game and self.game.is_db_stored:
-            ids = self.application.get_running_game_ids()
-            for game_id in ids:
-                if str(game_id) == str(self.game.id):
-                    return self.game
-            logger.warning("Game %s not in %s", self.game.id, ids)
+        for game in self.games:
+            if game and game.is_db_stored:
+                ids = self.application.get_running_game_ids()
+                for game_id in ids:
+                    if str(game_id) == str(game.id):
+                        return game
```

Comment:
If this class will support multiple games, this method should return a list, not one game or None.

This is because we'd really like to have a multi-select 'Stop' command, just below.